<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyOptionsWidget</class>
 <widget class="QWidget" name="ProxyOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>284</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rdbAutoProxy">
        <property name="text">
         <string>Automatic connection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbExplorerProxy">
        <property name="text">
         <string>Import proxy settings from Internet Explorer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbFireFoxProxy">
        <property name="text">
         <string>Import proxy settings from Mozilla Firefox</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbManualProxy">
        <property name="text">
         <string>Specify the proxy manually</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdtManualProxy" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lneProxyHost"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spbProxyPort">
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>1080</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chbProxyUserPassword">
          <property name="text">
           <string>Name and password is required to connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lneProxyUser">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lneProxyPassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lneProxyHost</tabstop>
  <tabstop>spbProxyPort</tabstop>
  <tabstop>chbProxyUserPassword</tabstop>
  <tabstop>lneProxyUser</tabstop>
  <tabstop>lneProxyPassword</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rdbManualProxy</sender>
   <signal>toggled(bool)</signal>
   <receiver>wdtManualProxy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbProxyUserPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>lneProxyUser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chbProxyUserPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>lneProxyPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
